app/demo-loop/Transporter.cc

```diff
@@ -163,14 +165,9 @@ void launch_models(TransporterInput const& host_params, CoreRef<M> const& data)
     // Loop over physics models IDs and invoke `interact`
     for (auto model_id : range(ModelId{host_params.physics->num_models()}))
     {
-        ProcessId      pid  = host_params.physics->process_id(model_id);
-        const Process& proc = host_params.physics->process(pid);
-        if (proc.type() != ProcessType::electromagnetic_msc)
-        {
-            // Do not launch interact if the process is electromagnetic_msc
-            const Model& model = host_params.physics->model(model_id);
-            model.interact(data);
-        }
+        // Do not launch interact if the process is electromagnetic_msc
+        const Model& model = host_params.physics->model(model_id);
+        model.interact(data);
```

Comment:
We don't need this function anymore, right?

Reply:
oh crap, that's launching an extra set of nullop kernels at every step 😂 nice catch!